CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-Next
+0.4.0 (02/16/2017)
 ----
 
-* Your contribution here.
+* [#21](https://github.com/ruby-grape/grape-roar/pull/21): Fixes serialization issue due to [representable](https://github.com/trailblazer/representable) API change. - [@mach-kernel](https://github.com/mach-kernel)
```

Comment:
The syntax of these things is not to have a period in the end ;) 

Add the name below.